packages/react-urql/examples/1-getting-started/package.json

```diff
@@ -15,11 +15,11 @@
   },
   "dependencies": {
     "@urql/devtools": "latest",
-    "graphql": "^14.6.0",
+    "graphql": "^15.0.0",
     "graphql-tag": "^2.10.3",
     "idb": "^5.0.1",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
-    "urql": "latest"
+    "urql": "link:../.."
```

Comment:
You managed to get this working? Educate me 😅 

I always have issues with conflicting react versions and end up having to fight with links.

Reply:
On second thought, we have this [running in codesandbox](https://codesandbox.io/s/github/FormidableLabs/urql/tree/master/packages/react-urql/examples/1-getting-started) so this might break usage. In the past I've used a sketchy sed script to switch to "link" for development usage but I think there might be a better way

Reply:
I think my changes break it in codesandbox. Debugging...

Reply:
No worries, my guess would be that it's this line - the codesandbox root directory is the example folder so relative urls (such as in this link) outside this folder will cause issues.